src/tesClient/messages/MessageResponder.js

```diff
@@ -21,6 +22,7 @@ class MessageResponder {
                         messageBodyContents
                     } =
                         MessageParser.parseMessage({ message });
+                    console.log('in parse: ' + incomingRequestId + messageBodyType + messageBodyContents);
```

Comment:
You can use [debug module](https://www.npmjs.com/package/debug) to avoid console.log in production.